.github/workflows/test-nightly.yml

```diff
@@ -43,18 +43,11 @@ jobs:
           key: ${{ matrix.build }}-cargo-index-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: |
             ${{ matrix.build }}-cargo-index-
-      - name: Cache Cargo build
-        uses: actions/cache@v1
```

Comment:
Q: Hm, I saw the commit message saying you can't cache this when testing locally, but I'm not sure why? How is this different than my local machine reusing `target` and running tests again locally?

Reply:
on nightly if i cache but the word nightly, i'll end up trying to use a crate compiled with days old rust (since the version changes... nightly)

Reply:
Ah, gotcha. FWIW I've used [actions-rs](https://actions-rs.github.io/) [here](https://github.com/apollographql/rust/pull/84) -- it outputs a toolchain hash (via `steps.toolchain.outputs.rustc_hash`) which changes nightly, so the cache is consistent with the version.

I'd also recommend using cache@v2 for DRYer workflows, but that can be done separately.

Reply:
yeah, i may consider it but for another PR!